src/nls/en/strings.json

```diff
@@ -94,7 +94,11 @@
                 "HEADLINE": "Design Space",
                 "BODY_FIRST": "Twitter",
                 "BODY_SECOND": "GitHub",
-                "BODY_THIRD": "Forum"
+                "BODY_THIRD": "Forum",
+                "BODY_ADVERTISEMENT_1": "You seem like the type who likes to check out betas. We've been hard at work building a new end-to-end solution for UX designers, including great integration with Photoshop. We're happy to share that",
+                "BODY_ADVERTISEMENT_2": "Adobe Experience Design CC",
+                "BODY_ADVERTISEMENT_3": "is now in public beta.",
```

Comment:
Those fragments are pulled out because they have HTML tags wrapped around them. I'm not actually sure how to have a single localized string in React that includes embedded tags. If you have a solution in mind, can you break it down for me a little more?